Dockerfile

```diff
@@ -1,56 +1,36 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
```

Comment:
Do we want to wait until all the prow-based jobs, which rely on this Dockerfile and we substitute images based on known base image names, until after they all have been migrated to GH actions?

Reply:
Yeah, same comment w.r.t to the base.Dockerfile, which is listed as the build root Dockerfile in the o/release prow configuration. I'm working towards removing the prow tests after I get the marketplace k8s bump off my plate.

Reply:
Definitely, let me know if I can help with that.

/hold

Reply:
@benluddy I have a PR open for adding the unit/verify tests to GH actions in https://github.com/operator-framework/operator-lifecycle-manager/pull/2096 and once that merges we can merge the [o/release PR](https://github.com/openshift/release/pull/18324) for removing these OCP specific Dockerfile(s).

Reply:
Both of those aformentioned PRs merged - I'm on the fence whether we want to just remove the OCP-specific Dockerfile(s) here (e.g. Dockerfile, base.Dockerfile) and keep the upstream.Dockerfile w.o. deflating/merging with a single Dockerfile. I think it's fine to iterate on this going forward as our upstream CI doesn't test using the same container image we attempt to successfully build.

Reply:
> just remove the OCP-specific Dockerfile(s) here

That's all that is happening here -- Dockerfile and base.Dockerfile are removed, and upstream.Dockerfile is renamed to Dockerfile with no changes. The diff is just misleading.